api/v6/report_server.thrift

```diff
@@ -116,7 +116,8 @@ struct RunData {
   4: i64                       duration,     // Duration of the run (-1 if not finished).
   5: i64                       resultCount,  // Number of results in the run.
   6: string                    runCmd,       // The used check command.
-  7: map<DetectionStatus, i32> detectionStatusCount // Number of reports with a particular detection status.
+  7: map<DetectionStatus, i32> detectionStatusCount, // Number of reports with a particular detection status.
+  8: string                    versionTag    // Version tag of the latest run.
```

Comment:
❗️ ❗️ Please be **very careful**about changing the fields of a `struct`. Thrift supports an innate "versioning" for adding new member fields. Please check **extensively** if an older client can still parse this new, modified version of the struct!

Reply:
I tested it with a 6.0 client by calling the `CodeChecker cmd runs`